### ISSUE TITLE
Native modules on linux (and likely also mac)

### ISSUE THREAD

**Issue by Hoi15A** [AirFusion45/Power-Bot#83]
Came across this repo and noticed this in your README.md
> Open up a PowerShell in administrator mode, then run the command: npm i -g --production windows-build-tools NOTE: Make sure the installation is fully complete before you close the PowerShell window! <----- This will only work on windows computers... **Someone fill me in via the issues tab on how to do this on mac or linux.**

You don't need to do anything on Linux at least. Node will happily just compile native modules (at least on Linux which is what I tried it on) thus that package is not needed.

<hr>

Quick tl;dr on how I installed and ran it on my PC running Arch Linux with NodeJS `11.3.0`
```sh
git clone https://github.com/AirFusion45/Power-Bot.git
cd Power-Bot
npm i
cp config\ \(example\).json config.json; vim config.json # Edit the config
node .
```

**Comment by ethamitc** [comment]
Yes this package isnt needed on linux, but that's because it natively has the build tools. Windows dosnt, therefore the package is needed to build on windoss

**Comment by AirFusion45** [comment]
okay, it seems like the package is not needed on mac too. I tried installing on mac, it threw an error, but the bot ran without issues. So I will just add a mac/linux note? I guess... Or have a mac/linux version? but thx for letting me know!

**Comment by intrnl** [comment]
- Debian/Ubuntu based distributions   
`sudo apt install build-essential`
- Fedora/RHEL based distributions   
`sudo dnf install gcc-c++ make`, use `yum` if you don't have `dnf`

**Comment by intrnl** [comment]
[Instructions on getting native modules to build on macOS](https://github.com/nodejs/node-gyp/blob/master/README.md#on-macos)

**Comment by Hoi15A** [comment]
Wouldn't node pull in those build tools as dependencies tho?

I don't recall ever having to install build-essential on Debian 9

**Comment by intrnl** [comment]
The build tools are optional, the reason that you didn't need to install the build packages is probably because you've installed them before or Debian already comes with it.

**Comment by Hoi15A** [comment]
I see, maybe just a note that you might have to install build tools on linux and macOS along with a link or two to node's documentation on that would probably be more than enough for the Readme here I suppose
